File: IT/1/Funzioni/Esercizi/Python/esercizio4.py
# file esercizio4.py

from __future__ import print_function
from sys import stdout
from os import linesep

def abs(n):
	return n if n >= 0 else -n

for i in range (-7, 8):
	print(abs(i), end=" ")

stdout.write(linesep)

File: IT/1/Funzioni/Esercizi/C++/esercizio4.cpp
/* file esercizio4.cpp */

#include <iostream>

using std::cout;
using std::endl;

int abs(int n) {
	return n >= 0 ? n : -n;
}

int main() 
{
	int i = -7;
	while (i <= 7) {
		cout << abs(i) << " ";
		i++;
	}

	cout << endl;
}

